src/components/CreateProposal/CreateProposalContainer.tsx

```diff
@@ -108,6 +108,8 @@ class CreateProposalContainer extends React.Component<IProps, null> {
 
   public render() {
     const { dao } = this.props;
+    const proposalDescriptions = dao.proposals.map((proposal) => (proposal as IProposalState).description);
```

Comment:
we only need to prevent duplicates of open proposals, so if proposal has been executed or failed its ok to have another with the same description

Reply:
That means descriptions are not strictly unique, how would the server know which one to return?

Reply:
we use proposalId when possible, otherwise we switch to description hash. this lets people create  new proposal from the same Google Docs link if it fails the first time, for example

Reply:
fixed at https://github.com/daostack/alchemy/pull/102/commits/9d39e911117f97d893b4d442b18a0798261ad05c

is using `proposalEnded` ok for this?